api/auth/auth.cpp

```diff
@@ -231,48 +231,40 @@ error::Error FetchJWTToken(
 }
 
 void Authenticator::ExpireToken() {
-	unique_lock<mutex> lock {auth_lock_};
```

Comment:
Nit: Would perhaps say, "The code is run in the event loop, **not in a separate thread**", just to be extra clear.